test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -480,23 +480,24 @@ public static MockServer CreateMockServer(IList<IPackage> packages)
         /// Path to nuget.exe for tests.
         /// </summary>
         public static string GetNuGetExePath()
-        {
-            return _nuGetExePath.Value;
-        }
-
-        private static readonly Lazy<string> _nuGetExePath = new Lazy<string>(GetNuGetExePathCore);
-
-        private static string GetNuGetExePathCore()
         {
             var targetDir = ConfigurationManager.AppSettings["TestTargetDir"] ?? Directory.GetCurrentDirectory();
-            var nugetexe = Path.Combine(targetDir, "NuGet", "NuGet.exe");
+            var fileName = "NuGet.exe";
+            var nugetexe = Path.Combine(targetDir, "NuGet", fileName);
+            // Revert to parent dir if not found under layout dir.
+            if (!File.Exists(nugetexe)) nugetexe = Path.Combine(targetDir, fileName);
```

Comment:
Add curly brackets.

Reply:
> Braces may be omitted only if the body of every block associated with an `if`/`else if`/.../`else` compound statement is placed on a single line.

It says this in the [coding guidelines](/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md) doc. What should I do?